js-functions/wk1 submission.js

```diff
@@ -0,0 +1,40 @@
+/*
+Example x = 'abcde';
+1. Write a JavaScript function that reverse a string.
+Expected Output : 'edcba'
+Click me to see the solution
+
+2. write a function that returns a grade based on the followin conditions
+
+
+If above 90, then A .
+else If above 80, then B .
+else If above 70, then C .
+else If above 60, then D .
+else F .
+
+*/
+
+function reverse(str){
+  var newString = ''
+  for (var i = str.length - 1; i >= 0; i--) {
```

Comment:
there are multiple ways to reverse a string. such as

```
function reverseString(str) {
    // Step 1. Use the split() method to return a new array
    var splitString = str.split(""); // var splitString = "hello".split("");
    // ["h", "e", "l", "l", "o"]
 
    // Step 2. Use the reverse() method to reverse the new created array
    var reverseArray = splitString.reverse(); // var reverseArray = ["h", "e", "l", "l", "o"].reverse();
    // ["o", "l", "l", "e", "h"]
 
    // Step 3. Use the join() method to join all elements of the array into a string
    var joinArray = reverseArray.join(""); // var joinArray = ["o", "l", "l", "e", "h"].join("");
    // "olleh"
    
    //Step 4. Return the reversed string
    return joinArray; // "olleh"
}
 
```

from https://www.freecodecamp.org/news/how-to-reverse-a-string-in-javascript-in-3-different-ways-75e4763c68cb/